### ISSUE TITLE
Add explicit support for comments of all policy rules, whitelist items, and mapping rules

### ISSUE THREAD

**Issue by zhill** [anchore/anchore-engine#187]
To help aid policy context and carry information for policy authors to users consuming policy and/or policy results, add support for comments or description elements in the bundle.

Items needing added "comment" field:

- Policy rules
- Whitelist items
- Mapping rules

Currently, the spec is loose enough to allow extra json content to be stored, but this should be standardized into specific fields and the fields restricted on the bundle to ensure only valid fields are persisted for stability.

**Comment by nbrys** [comment]
+1 for this feature request

**Comment by KnoxAnderson** [comment]
We're getting this request often - what are you seeing on your side?

**Comment by holt** [comment]
@KnoxAnderson yes, in response to this we've provided the ability to add comments to rules and whitelists in the UI for a while now (for the last couple of major releases I think).
